### ISSUE TITLE
Screen and Window sharing not working on X

### ISSUE THREAD

**Issue by polpoctopus** [mdouchement/docker-zoom-us#34]
Hi! The zoom application doesn't see any other window running in X. That's really strange because I tried inside the container to install and launch chromium from the zoom user and it could be able to capture other X11 windows' content without any problem. So it doesn't seem a matter of permissions, and #20 is already taken into account. Host is Archlinux, kernel 5.9.1 and I am using Sway on Wayland.
![2020-10-31_14-10-1604151965](https://user-images.githubusercontent.com/73746790/97780903-dd2a3d80-1b87-11eb-8552-0100b22e21b7.jpg)

**Comment by mdouchement** [comment]
I don't know if Sway on Wayland is causing the issue. Can you provide some logs? https://github.com/mdouchement/docker-zoom-us#troubleshooting
_If the logs folder is not reachable from you host, you may able to reach it from the container._

**Comment by polpoctopus** [comment]
I don't believe Sway is causing any problem, as screen sharing is perfectly working on e.g. a teams video call running on chromium from the zoom user inside the container as you can see from here
![2020-11-01_01-11-1604190979](https://user-images.githubusercontent.com/73746790/97792747-b1887100-1be2-11eb-89e3-3942d9ed39a8.jpg)

Logs are:

`% sudo docker logs zoomus
Adding user `zoom' to group `sudo' ...
Adding user zoom to group sudo
Done.
zoom
Another composite manager is already running (wlroots wm)`

and
the only file in ${ZOOM_HOME}/.zoom/logs is:
[zoom_stdout_stderr.log](https://github.com/mdouchement/docker-zoom-us/files/5470036/zoom_stdout_stderr.log)

**Comment by mdouchement** [comment]
Not much details in stdout logs :/
Maybe somthing inside the container in the following directories:
- `/var/log`
- `~/`
- `~/.config`

But zoomus on Linux should output logs in its `.zoom/logs` directory.

**Comment by polpoctopus** [comment]
Sure!
Files in `/var/log` are:
* `/var/log/btmp` is empty
* `/var/log/alternative.log`  [alternatives.log](https://github.com/mdouchement/docker-zoom-us/files/5488574/alternatives.log)
* `/var/log/dpkg.log`[dpkg.log](https://github.com/mdouchement/docker-zoom-us/files/5488575/dpkg.log)
* `/var/log/faillog` is empty
* `/var/log/fontconfig.log` [fontconfig.log](https://github.com/mdouchement/docker-zoom-us/files/5488576/fontconfig.log)
* `/var/log/lastlog` is empty
* `/var/log/wtmp` is empty

In the home directory:
* `~/.config/zoomus` is empty
* `~/.zoom/logs/zoom_stdout_stderr.log` is nonempty, identical to reply above
* `~/.zoom/.zoom` directory is empty

That's so strange, I also tried from firefox running inside the container, opened jitsi call and could share content of x windows without any problem
